weave/trace_server/sqlite_trace_server.py

```diff
@@ -1531,9 +1540,21 @@ def threads_query_stream(
             )
 
     def evaluate_model(self, req: tsi.EvaluateModelReq) -> tsi.EvaluateModelRes:
-        raise NotImplementedError(
-            "evaluate_model is not implemented for SQLite trace server"
+        if self._evaluate_model_dispatcher is None:
+            raise ValueError("Evaluate model dispatcher is not set")
+        if req.wb_user_id is None:
+            raise ValueError("wb_user_id is required")
```

Comment:
Little confused on the wb_user_id check

Seems like the only reason it would be optional on EvaluateModelReq would be if we had automated dispatching which isn't attributed to a user, but maybe a project or team.

if that's the case though, why is is required here?

Could we just make this field non optional?